src/features/tokens/hooks.ts

```diff
@@ -69,22 +69,24 @@ export function getInitialTokenIndex(
   originQuery?: string,
   destinationQuery?: string,
   defaultOriginToken?: Token,
+  defaultDestinationChain?: string,
 ): number | undefined {
   const firstToken = defaultOriginToken || warpCore.tokens[0];
-  const connectedToken = firstToken.connections?.[0];
+  const connectedToken = firstToken.connections?.[0].token;
```

Comment:
very nit but i find this diff a little hard to review because while you are correct that probably the old name was not specific, i am looking for functionality changes but all that i can see if making the statement below slightly simpler